code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -313,7 +313,7 @@
 		if(istype(the_reagent, /datum/reagent/medicine))
 			medibonus += 1
 	M.adjustToxLoss(-0.5 * min(medibonus, 3)) //not great at healing but if you have nothing else it will work
-	M.adjustOrganLoss(ORGAN_SLOT_LUNGS, 0.5) //kills at 40u
+	//M.adjustOrganLoss(ORGAN_SLOT_LUNGS, 0.5) //SKYRAT EDIT - No, cease to exist.
```

Comment:
This still edits and removes all risk of using the chemical.

Reply:
Yes, because Multiver is literally just renamed Charcoal with a tacked on damaging aspect for literally no reason.